### PR TITLE
[Feature] add NavBar component

### PR DESCRIPTION
**Pull request by jomadoye**
This PR adds the Navigation component
A screen shot is attached below.
screen shot 2017-07-31 at 2 15 50 am

### CODE REVIEW

**Comment by BolajiOlajide** [comment]
@kingisaac95 Have you reached a final decision on the color scheme to use?
I need a response before this can be merged.

File: client/components/NavigationBar/NavigationBar.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export class NavigationBar extends React.Component {
+  render() {
+    return(
+      <nav className="cyan darken-3">
+        <div className="nav-wrapper">
+          <a href="#" className="brand-logo">&nbsp; OneLaptop</a>
+          <ul id="nav-mobile" className="right hide-on-med-and-down">
+            <li><a id="NavBarText" href="#">Home</a></li>
```
Comment (BolajiOlajide) [inline client/components/NavigationBar/NavigationBar.jsx]: can the idea be lower case?

Comment (BolajiOlajide) [reply client/components/NavigationBar/NavigationBar.jsx]: they can be called `navbar-text`

File: client/components/NavigationBar/NavigationBar.test.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+describe('NavigationBar', () => {
+  const wrapper = setup();
+  const menuItems = ['Home','About','Donate','Contact us'];
+
+  it('renders the name of the application', () => {
+    expect(wrapper.find('.brand-logo')
+          .props()
+          .children)
+        // .toEqual('  OneLaptop');
```
Comment (BolajiOlajide) [inline client/components/NavigationBar/NavigationBar.test.js]: Kindly remove this comment since it's not needed

File: client/index.html [hunk 3]
```diff
@@ -2,10 +2,12 @@
 <html>
   <head>
     <script src="//d2wy8f7a9ursnm.cloudfront.net/bugsnag-3.min.js" data-apikey="609d1f020472985e992a1d3d920f66eb"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/css/materialize.min.css">
     <title> One-Laptop </title>
   </head>
   <body>
     <div id="app"></div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/js/materialize.min.js"></script>
```
Comment (BolajiOlajide) [inline client/index.html:10]: can you do us the favor of importing `jquery` too?

**Review by BolajiOlajide — COMMENTED** [review]
Good work Jed!

**Comment by andela-oakinseye** [comment]
Good job Jed!

File: client/components/NavigationBar/NavigationBar.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export class NavigationBar extends React.Component {
+  render() {
+    return(
+      <nav className="cyan darken-3">
+        <div className="nav-wrapper">
+          <a href="#" className="brand-logo">&nbsp; OneLaptop</a>
```
Comment (kingisaac95) [inline client/components/NavigationBar/NavigationBar.jsx]: I notice you're using &nbsp; and a space in the brand logo. Please refactor it and use padding instead for consistency.

File: client/index.html (hunk 3 above)
Comment (kingisaac95) [reply client/index.html:10]: I have imported it locally. It'll reflect with my PR. Is that okay with you @andela-jomadoye?

**Comment by jomadoye** [comment]
@BolajiOlajide All changes have been made.

**Review by BolajiOlajide — APPROVED** [review]
This looks good and ready to be merged.

**Pull request merged**